pages/accordion_page.ts

```diff
@@ -0,0 +1,26 @@
+import { Page, expect, test } from '@playwright/test';
+
+export class AccordionPage {
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+
+    async open() {
+        await this.page.goto('https://webdriveruniversity.com/Accordion/index.html');
+    }
+
+    async waitForLoadedComplete() {
+        await test.step ('Wait for complete loading.', async () => {
+            await expect(this.page.locator('#hidden-text')).toHaveText('LOADING COMPLETE.',{timeout: 20000});
+        })
+    }
+
+    async verifyTextInAccordionTimeout() {
+        await test.step ('Verify text in last accordion.', async () => {
+            await expect(this.page.locator('#timeout')).toHaveText("This text has appeared after 5 seconds!");
```

Comment:
how do you know that we're waiting 5 seconds here? even if it's default timeout it may be changed any time

Reply:
In my task: "Check and verify the text in the last accordion." - This was only verifying the text in the accordion.